catalog/internals/webpack/webpack.prod.babel.js

```diff
@@ -147,4 +44,20 @@ module.exports = require('./webpack.base.babel')({
   performance: {
     assetFilter: (assetFilename) => !/(\.map$)|(^(main\.|favicon\.))/.test(assetFilename),
   },
+
+  // options passed to all instances of HtmlWebpackPlugin
+  html: {
+    minify: {
```

Comment:
`build` directory has the same size with just `minify: true`. I don't think this fine-tuning worth it

Reply:
looks like we can omit this altogether and it will still minify the output in production mode

Reply:
:heavy_plus_sign: